homeassistant/components/cert_expiry/sensor.py

```diff
@@ -67,6 +67,11 @@ def name(self):
         """Return the name of the sensor."""
         return self._name
 
+    @property
+    def unique_id(self):
+        """Return a unique id for the sensor."""
+        return f"cert_expiry-{self.server_name}-{self.server_port}"
```

Comment:
We don't need to prefix with the integration domain. The entity registry is aware of both the integration domain and the platform domain.

Reply:
Thanks, I wasn't aware.